xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5156,7 +5156,7 @@ xe.blog.migration.inspace=in space
 xe.blog.migration.skipping=Skipping protected page
 xe.blog.migration.done=Done.
 xe.blog.migration.backtoblog=Back to the blog
-xe.blog.migration.pleaseconfirm=Please confirm that you want to migrate old articles to the new blog application:
+xe.blog.migration.pleaseconfirm=Please confirm if you want to migrate old articles to the new blog application:
```

Comment:
I'm not convinced that "if" is better than "that" here. I find it less good actually... Can someone confirm it's correct?

Reply:
I also prefer `that`, but neither ones are technically wrong. It just depends on the nuance you give it. Both practically end up in the same place, just that `that` asks for your intention, while `if` asks for your permission. Both ask you to confirm and I consider both acceptable.

Reply:
Then why accept the new one?